source/Octopus.Cli/Commands/Deployment/ListDeploymentsCommand.cs

```diff
@@ -78,7 +79,10 @@ await Repository.Deployments
                 deploymentResources[item].ReleaseResource = await Repository.Releases.Get(item.ReleaseId).ConfigureAwait(false);
 
                 if (!string.IsNullOrEmpty(item.ChannelId))
-                    deploymentResources[item].ChannelResource = await Repository.Channels.Get(item.ChannelId).ConfigureAwait(false);
+                {
+                    var projectForDeployment = projectResourcesById[item.ProjectId];
```

Comment:
Fortunately already have the project details, so no need to go and fetch them again.